src/inc/CombFilter.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+//IIR class CCombFilterBase;
+#include <math.h>
+#include <algorithm> 
+#include <complex>
+using namespace std;
+#include "CombFilterIf.h"
```

Comment:
You don't need to include the combfilterIf.h here, as combfilter interface contains the user functionality which will then point to combfilter base for the particular filter processes.